README.md

```diff
@@ -1,30 +1,130 @@
 # NotionDiscordIntegration
 
-Notion Integration Into Discord
+A Discord bot that integrates with Notion for updates and notifications.
 
-### How to Deploy:
+## Setup
 
-**Setup**
+1. Clone the repository or copy the necessary files:
+   ```bash
+   git clone https://github.com/BloomGameStudio/NotionDiscordIntegration.git
+   ```
 
-1. Do one of the following:
+2. Create and configure the environment variables:
+   - Copy `.env.example` to `.env`
+   - Fill in the required values:
+     ```
+     DISCORD_BOT_TOKEN=your_discord_bot_token
```

Comment:
this shouldn't be included either. it makes it so we have to maintain the info in two places